src/ResourceManager/Reservations/Commands.Reservations.Test/ScenarioTests/ReservationTests.ps1

```diff
@@ -0,0 +1,205 @@
+# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+$subscriptionId ="98df3792-7962-4f18-8be2-d5576f122de3"
+$reservationOrderId = "55793bc2-e5c2-4a98-9d5c-0a0bce6cf998"
+$reservationId = "55ef9160-825e-4ca3-969e-c919de3c57ec"
+
+<#
+.SYNOPSIS
+Get reservation catalog
+#>
+function Test-GetCatalog
+{
+	$catalog = Get-AzureRmReservationCatalog -SubscriptionId $subscriptionId
+	Foreach ($item in $catalog)
+	{
+		Assert-NotNull $item.ResourceType
```

Comment:
This seems like it is not enogh actual data to test the api - what should the values be for each of these things - this should be repeated across the board.

Reply:
tests are now more reflective of actual fields